Jenkinsfile

```diff
@@ -6,11 +6,12 @@ pipeline {
         buildDiscarder(logRotator(numToKeepStr:'25'))
     }
     triggers {
-        cron('H H(20-23) * * *')
+        cron('H H(17-19) * * *')
```

Comment:
I adjusted this trigger to avoid overlapping this job with others pipelines (Alliance). This may have been a contributor to some of the mismatched/missing artifact issues that caused build failures.